frontend/src/metabase/components/LastEditInfoLabel/LastEditInfoLabel.jsx

```diff
@@ -38,23 +39,40 @@ function formatEditorName(lastEditInfo) {
   return name || lastEditInfo.email;
 }
 
-function LastEditInfoLabel({ item, user, onClick, className }) {
+function LastEditInfoLabel({
+  item,
+  user,
+  prefix = t`Edited`,
+  onClick,
+  className,
+}) {
   const lastEditInfo = item["last-edit-info"];
   const { id: editorId, timestamp } = lastEditInfo;
-  const time = moment(timestamp).fromNow();
+
+  const momentTimestamp = moment(timestamp);
+  const timeLabel =
+    timestamp && momentTimestamp.isValid() ? momentTimestamp.fromNow() : null;
 
   const editor = editorId === user.id ? t`you` : formatEditorName(lastEditInfo);
+  const editorLabel = editor ? t`by ${editor}` : null;
+
+  const label =
+    timeLabel || editorLabel
+      ? [timeLabel, editorLabel].filter(Boolean).join(" ")
+      : null;
 
-  return (
-    <Tooltip tooltip={<DateTime value={timestamp} />}>
+  return label ? (
+    <Tooltip tooltip={<DateTime value={timestamp} />} isEnabled={!!timeLabel}>
       <TextButton
         size="small"
         className={className}
         onClick={onClick}
         data-testid="revision-history-button"
-      >{t`Edited ${time} by ${editor}`}</TextButton>
+      >
+        {t`${prefix} ${label}`}
```

Comment:
Not sure if `t` is needed here - both `prefix` and `label` should come translated already.